crates/nu-command/tests/commands/complete.rs

```diff
@@ -1,12 +1,24 @@
 use nu_test_support::nu;
 
 #[test]
-fn basic() {
-    let actual = nu!(r#"
-        (^echo a | complete) == {stdout: "a\n", exit_code: 0}
+fn basic_stdout() {
+    let without_complete = nu!(r#"
+        ^echo a
```

Comment:
we're not supposed to be using `^echo` in our tests because all echoes do not work the same on all operating systems. this is why `nu --testbin cococo` was created.
https://github.com/nushell/nushell/blob/42bb42a2e1718469133fe3a71ac76c40389bbb30/src/test_bins.rs#L60-L75

Reply:
Oh, it's very nice. I've been looking for something like it.
I'll replace with it.